### ISSUE TITLE
GPU usage is first super high then low, anyone who has a fix?

### ISSUE THREAD

**Issue by nicolai256** [NVlabs/stylegan3#141]
**Describe the bug**
when the command is computing the stylegan3 in the beginning it uses a lot of cpu but after it only uses half of that..
when i put `--batch=32gb --batch-gpu=16` it says out of memory but i'm working with 2x RTX A6000 that have 48gb
so i put in `--batch=16 --batch-gpu=8` to be able to start the training, now that it's training it says reserved 34GB of memory..
is there any way to make it use all memory available? (i'm renting and it's kind of a waste..)

**Expected behavior**
make stylegan3 use all gpu ram available

**Desktop (please complete the following information):**
 - OS: [Linux Ubuntu 20.04]
 - PyTorch version ( pytorch 1.9.0)
 - CUDA toolkit version (CUDA 11.4)
 - NVIDIA driver version
 - GPU [2xA6000]

**Comment by PDillis** [comment]
Not really, it's super high at the beginning and usually, it also goes up and down during training. That's why it's best to leave a bit of a buffer and not use 100% of the GPU capacity. Which config and image resolution are you using? `config-r` is super expensive and slower to train and, if you don't really need the rotational equivariance, then it's a waste, so consider `config-t` or `stylegan2` if the results with those configs are good enough for you.

**Comment by PDillis** [comment]
No, you can't transfer from `config-r` to `config-t`. Those 12 hours will be nothing compared to how slow `config-r` is to train vs `config-t`, so now's the time to change. Are you starting from a pre-trained pkl? There are some available and, if you're using my repo, you can use some that I've found [here](https://github.com/PDillis/stylegan3-fun/blob/4ce9d6f7601641ba1e2906ed97f2739a63fb96e2/torch_utils/gen_utils.py#L401). Just note that the final Wombo ones don't really work, so I have to remove them. The way to use these is to use the command `--config=stylegan3-t --resume=afhq512` to use the Animal Faces HQ (512x512 resolution), for example. Otherwise you can download locally the models and just give the path.

Lastly, what augmentations are you using? What value of `--gamma`? How many images do you have? The [StyleGAN2-ADA paper](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada/ada-paper.pdf) is extremely useful to know some of the values to use, so I recommend you read that as you train your models (or before, whatever you choose).
